games/godot/Portfile

```diff
@@ -25,11 +25,11 @@ long_description    Godot Engine is a cross-platform game engine for \
 if {$subport eq ${name}} {
     if {${os.major} <= 15} { PortGroup legacysupport 1.1 }
```

Comment:
this should ideally also check for the platform as in `{if ${os.platform} eq "darwin"` && .....`}

Reply:
I've always wondered about that.... Doesn't the `platforms darwin` line already imply that the entire file is for `${os.platform} eq "darwin"`? Or am I misunderstanding the purpose of `platforms darwin`?

(and by the way, I do use `${os.platform} eq "darwin"` most other places. I've just never understood why; I simply copied what I was seeing in other Portfiles)

Reply:
please just add it, if you have it everywhere else in the Portfile and other ones use it as well (and it's always requested if not present when checking `os.major`). I am not 100% sure about it, but it doesn't hurt....

Reply:
I've gone ahead and made the requested change, but I also asked the question on the dev mailing list. It's interesting that the conversation seems to have brought up the possibility that wrapping chunks of code inside of a `platform darwin {` could eliminate the need to check for `${os.platform} eq "darwin"` each and every time that `${os.major}` is used.